### PR TITLE
Mjm/powered by

### PR DESCRIPTION
**Pull request by mmcfarland**
## Overview

Adds several UI components:
* "Sponsored By" link and logo to IDB
* "Something Wrong?" link to the iD editor
* Preference for starting map at context of Georgetown

Connects #28 
Connects #27 

### Demo

screenshot from 2018-12-03 13 19 37

### Notes

These differ slightly from the precision of the wireframes, but I believe the implement to be sufficient and expedient.

screenshot from 2018-12-03 13 22 40

## Testing Instructions

 * Click IDB link and confirm it opens in new tab
 * Navigate the map, and confirm that the iD editor is opened at the correct extent (you may need an account to see this fully)

### CODE REVIEW

**Comment by kellyi** [comment]
Taking a look at this!

File: src/js/src/components/SponseredByControl.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Control from 'react-leaflet-control';
+import { oneOf } from 'prop-types';
+
+import { controlPositionsEnum } from '../constants';
+import idbLogo from '../../../images/logo-idb-full.png';
+
+export default function SponseredByControl({ position }) {
```
Comment (kellyi) [inline src/js/src/components/SponseredByControl.jsx]: Type in component name -> `SponseredByControl` -> `SponsoredByControl`

File: src/js/src/components/SponseredByControl.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    return (
+        <Control position={position}>
+            <div className="logo -idb">
+                <div>sponsered by</div>
```
Comment (kellyi) [inline src/js/src/components/SponseredByControl.jsx]: Typo: `sponsered by` -> `sponsored by`

File: src/js/src/components/SomethingWrongControl.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+export default function SomethingWrongControl({
+    position,
+    getZoomAndCenter,
+}) {
+    return (
+        <Control position={position}>
+            <div className="something-wrong">
+                Something look wrong?&nbsp;
+                <button
+                    onClick={() => navigateToEditor(getZoomAndCenter())}
```
Comment (kellyi) [inline src/js/src/components/SomethingWrongControl.jsx]: This works well, so it's probably fine to leave as is. However, it occurred to me that all of the `window.open` logic could potentially be moved up into/in place of the `getCenterAndZoom` method bound to the class above.

Comment (mmcfarland) [reply src/js/src/components/SomethingWrongControl.jsx]: Seems low-consequence, but I like having the logic of the action be contained in the component and the map just providing the context it needs to do it.  I'm going to keep this as-is.

**Comment by mmcfarland** [comment]
I clearly can't spell, thanks for catching that.  Will merge when CI goes green again.

**Pull request merged**